db/migrate/20241225031242_populate_country_band_details.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class PopulateCountryBandDetails < ActiveRecord::Migration[7.2]
+  def change
```

Comment:
This should be `def up`. Then you can also create a `def down` which does `CountryBandDetail.delete_all` or similar.